state/dbstore.go

```diff
@@ -34,6 +36,7 @@ type Store interface {
 	Get(key string, i interface{}) (err error)
 	Put(key string, i interface{}) (err error)
 	Delete(key string) (err error)
+	Iterate(prefix string, iterFunc iterFunction) (err error)
```

Comment:
this should already be on master. please rebase

Reply:
Rebased.